datasets/mozetl/search_clients_daily/reference.md

```diff
@@ -0,0 +1,64 @@
+# Search Clients Daily
+
+<!-- toc -->
+
+# Introduction
+
+{% include "./intro.md" %}
+
+# Data Reference
+
+## Example Queries
+
+[This query](https://sql.telemetry.mozilla.org/queries/51141/source)
+calculates searches per `normalized_channel` for US clients on an arbitrary day.
+If you have trouble viewing this query,
+it's likely you don't have the proper permissions.
+For more details see the [search data documentation].
+
+
+## Scheduling
+
+This dataset is not currently scheduled.
+
+## Schema
+
+As of 2018-02-13, the current version of `search_clients_daily` is `v1`, and has a schema as follows:
+
```

Comment:
Could we add a comment indicating how far back it's currently been backfilled?

Reply:
Done